pds/contracts/IPackNFT.cdc

```diff
@@ -26,10 +26,6 @@ pub contract interface IPackNFT{
     /// This is emitted when owner of a PackNFT request for the entitled NFT to be
     /// deposited to its account
     pub event OpenRequest(id: UInt64)
-    /// New Pack NFT
```

Comment:
So there would be no Minted event? 🤔

Reply:
The issue was we made some "reckless" [changes](https://github.com/dapperlabs/platform-services/pull/599/) in the past, to make PackNFT to be more aligned with freshmint BlindNFT. But these changes make AllDay PackNFT in-compatible with our PDS anymore. After a discussion, we decided to remove the events and fields in IPackNFT to make AllDay PackNFT contract to be compatible. This is not the perfect solution. But the best we have right now.

Reply:
Minted event will still remain in the `PackNFT` contract. All events and fields removed from the `IPackNFT` will still remain in the `PackNFT`

Reply:
gotcha 👍  makes sense!